### PR TITLE
Add one more example of reducing null checks.

### LINKED ISSUE #9208: Clarify semantics of null-conditional operator in contexts expecting a boolean value.
For example, suppose you have an object `foo` with a boolean field called `IsBlah`. The following code in VB compiles...

```vb
If foo?.IsBlah Then DoSomething()
```

...but what exactly does it mean? I guess it is equivalent to this:

```vb
If foo IsNot Nothing AndAlso foo.IsBlah Then DoSomething()
```

...however none of this is spelled out in the documentation so it's not clear how this works. Is it like the code above or is the null-conditional operator returning a `Nullable(Of Boolean)` which is then handled in a special way by the `If` statement?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7427bc30-4d66-b234-4537-200d4419cc24
* Version Independent ID: 786cb51a-cd8d-95a7-a11c-c39245a01e9d
* Content: [Null-conditional Operators (Visual Basic)](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/operators/null-conditional-operators)
* Content Source: [docs/visual-basic/language-reference/operators/null-conditional-operators.md](https://github.com/dotnet/docs/blob/master/docs/visual-basic/language-reference/operators/null-conditional-operators.md)
* Product: **dotnet-visualbasic**
* GitHub Login: @KathleenDollard
* Microsoft Alias: **kdollard**

### PR DESCRIPTION
**Pull request by ericmutta**
Added an example of using the null-conditional operator to avoid checking for null when referencing a possibly-null object in an `If` statement (see discussion on #9208).

I wasn't sure where to slot in the extra example, hopefully the current position flows well with the rest of the page. 

PS: since this is the first doc change I have made that isn't a simple typo/grammar fix, I am very much open to the submission being further edited for "flow" or "tone". Much respect to the docs team, it's surprising how difficult it can be to write docs that are both thorough and easy to understand!

Fixes #9208

### CODE REVIEW

File: docs/visual-basic/language-reference/operators/null-conditional-operators.md [hunk 1]
```diff
@@ -33,6 +33,24 @@ If customers IsNot Nothing Then
 End If
 ```
 
+Sometimes you need to take an action on an object that may be null, based on the value of a boolean member on that object (like the boolean property `IsAllowedFreeShipping` in the code below):
```
Comment (rpetrusha) [inline docs/visual-basic/language-reference/operators/null-conditional-operators.md]:
```suggestion
Sometimes you need to take an action on an object that may be null, based on the value of a Boolean member on that object (like the Boolean property `IsAllowedFreeShipping` in the following example):
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for adding the additional example to the documentation, @ericmutta. Your additions looks good; I've left some minor suggestions.

**Comment by ericmutta** [comment]
@rpetrusha thanks for the suggested edit, I have applied it :+1:

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @ericmutta. I'll merge your PR now. The changes will appear live on docs.microsoft.com next Monday.

**Comment by ericmutta** [comment]
@rpetrusha many thanks, it's really fun to be able to contribute and see one's changes go live in a matter of days. Looking forward to doing more!

**Pull request merged**